releasetasks/templates/fennec/version_bump.yml.tmpl

```diff
@@ -0,0 +1,45 @@
+{% set buildername = "release-{}-{}_version_bump".format(branch, product) %}
```

Comment:
Have we changed the script to make sure we don't downgrade the version? It shouldn't be too hard.